Makefile.PL

```diff
@@ -2,67 +2,102 @@ use warnings;
+use ExtUtils::MakeMaker;
+our (%dev_requires, %runtime_suggests);
+(do './maint/Makefile.PL.include' or die $@) unless -f 'META.yml';
+
+my $eumm_version = eval $ExtUtils::MakeMaker::VERSION;
+my %eumm_args = (
+    NAME => 'DBIx::Class::Schema::Loader',
+    VERSION_FROM => 'lib/DBIx/Class/Schema/Loader.pm',
+    LICENSE => 'perl',
+    MIN_PERL_VERSION => '5.008001',
+    CONFIGURE_REQUIRES => {
+        'ExtUtils::MakeMaker' => '0',
+    },
+    TEST_REQUIRES => {
+        'DBD::SQLite'     => '1.29',
+        'Test::Exception' => '0.31',
+        'Test::More'      => '0.94',
+        'Test::Warn'      => '0.21',
+        'Test::Deep'      => '0.107',
+        'Test::Differences' => '0.60',
+        # temporary, needs to be thrown out
+        'DBIx::Class::IntrospectableM2M' => 0,
+        # core, but specific versions not available on older perls
+        'File::Temp'      => '0.16',
+        'File::Path'      => '2.07',
+    },
+    test => {TESTS => 't/*.t t/*/*.t'},
+    PREREQ_PM => {
+        'Carp::Clan'                  => 0,
+        'Class::Accessor::Grouped'    => '0.10008',
+        'Class::C3::Componentised'    => '1.0008',
+        'Class::Inspector'            => '1.27',
+        'Class::Unload'               => '0.07',
+        'curry'                       => '1.000000',
+        'Data::Dump'                  => '1.06',
+        'DBIx::Class'                 => '0.08127',
+        'Hash::Merge'                 => '0.12',
+        'Lingua::EN::Inflect::Number' => '1.1',
+        'Lingua::EN::Tagger'          => '0.23',
+        'Lingua::EN::Inflect::Phrase' => '0.15',
+        'List::Util'                  => '1.33',
+        'MRO::Compat'                 => '0.09',
+        'namespace::clean'            => '0.23',
+        'Scope::Guard'                => '0.20',
+        'String::ToIdentifier::EN'    => '0.05',
+        'String::CamelCase'           => '0.02',
+        'Sub::Util'                   => '1.40',
+        'Try::Tiny'                   => 0,
+        # core, but specific versions not available on older perls
+        'Digest::MD5'                 => '2.36',
+        'Exporter'                    => '5.63',
+    },
+    realclean => {
+        FILES => 'lib/DBIx/Class/Schema/Loader/Optional/Dependencies.pod'
+    },
+    EXE_FILES => [ qw(script/dbicdump) ],
+    META_MERGE => {
+        "meta-spec" => { version => 2 },
+        dynamic_config => 0,
+        resources => {
+            repository => {
+                type => 'git',
+                url => 'git@github.com/dbsrgits/dbix-class-schema-loader.git',
+                web => 'https://github.com/dbsrgits/dbix-class-schema-loader',
+            },
+            x_IRC => 'irc://irc.perl.org/#dbix-class',
+            license => [ 'http://dev.perl.org/licenses/' ],
+            x_MailingList => 'http://lists.scsys.co.uk/cgi-bin/mailman/listinfo/dbix-class',
+        },
+        no_index => {
+            directory => [qw(maint xt)],
```

Comment:
PAUSE already doesn't index anything in `inc`, `xt`, and `t`, so I think we should either list `t` explicitly here for completeness, or only explicitly list `maint` for conciseness.

Reply:
EUMM adds `t` and `inc`, so the list ends up sent to PAUSE as all 4. It is thus complete. The current code makes a META file with repetition.

Reply:
Having checked the current CPAN `META.yml`, it in fact doesn't have repetition. However it did locally when I added `t`, so I left it out here, rationale as above.